src/overlays/actors/ovl_En_Tana/z_en_tana.c

```diff
@@ -29,7 +29,7 @@ const ActorInit En_Tana_InitVars = {
     NULL,
 };
 
-static char* sShelfTypes[] = {
+static char* const sShelfTypes[] = {
     "木の棚", // "Wooden Shelves"
```

Comment:
youd generally see `const char*` not `char* const`, right

Reply:
Yeah this was a leftover from testing.  I'll take it out.